lib/matplotlib/text.py

```diff
@@ -771,19 +774,23 @@ def draw(self, renderer):
                 clean_line, ismath = textobj.is_math_text(line)
 
                 if textobj.get_path_effects():
-                    from matplotlib.patheffects import PathEffectRenderer
-                    textrenderer = PathEffectRenderer(
-                                        textobj.get_path_effects(), renderer)
+                    path_effects = textobj.get_path_effects()
+                    if textobj.get_usetex():
+                        ismath = "TeX"
+                    path, transform = renderer._get_text_path_transform(x, y, clean_line, prop, angle, ismath)
```

Comment:
True. I have modified the line to follow the PEP8 conventions.

-Michiel.

--------------------------------------------
On Wed, 6/17/15, Thomas A Caswell <notifications@github.com> wrote:

 Subject: Re: [matplotlib] Fix for issue4024 (#4532)
 To: "matplotlib/matplotlib" <matplotlib@noreply.github.com>
 Cc: "mdehoon" <mjldehoon@yahoo.com>
 Date: Wednesday, June 17, 2015, 1:07 PM
 
 In lib/matplotlib/text.py:
 > @@ -771,19 +774,23 @@ def
 draw(self, renderer):
 >                  clean_line, ismath =
 textobj.is_math_text(line)
 >  
 >                  if textobj.get_path_effects():
 > -                    from matplotlib.patheffects import
 PathEffectRenderer
 > -                    textrenderer = PathEffectRenderer(
 > -                                       
 textobj.get_path_effects(), renderer)
 > +                    path_effects =
 textobj.get_path_effects()
 > +                    if textobj.get_usetex():
 > +                        ismath = "TeX"
 > +                    path, transform =
 renderer._get_text_path_transform(x, y, clean_line, prop,
 angle, ismath)
 
 pep8 is choking on this line.
 
 —
 Reply to this email directly or view
 it on GitHub.